Lang/Spanish.pj.Lang

```diff
@@ -419,7 +420,7 @@
 //Edit Cheat
 #1027#  "Editar Truco"
 #1028#  "Actualizar Truco"
-#1029#  "El Truco ha sido cambiado.\n\n¿Deseas actualizar?"
+#1029#  "El Truco ha sido cambiado.\n\n¿Quieres actualizar?"

```

Comment:
That is not correct.
"Desear" has three possible meanings, that can be translated to English:
* To wish
* To desire
* To want

I just gave you a sample text that shows that verb is NOT limited to "desires", as nouns. Please don't disregard it.

I've studied and applied formal Spanish in many countries through many years (also a native speaker, BTW) I'm not making up these concepts.